packages/next/src/build/webpack/loaders/next-metadata-route-loader.ts

```diff
@@ -106,9 +106,9 @@ export async function GET(_, ctx) {
   const id = __metadata_id__[0]
   const imageMetadata = generateImageMetadata ? await generateImageMetadata({ params }) : null
   if (imageMetadata) {
-    const hasId = imageMetadata.some((item) => { item.id === id })
+    const hasId = imageMetadata.some((item) => item.id.toString() === id)
```

Comment:
Is there a case when `id` won't exist?

Reply:
I think we can just throw here id isn’t existing?

Reply:
yes, gonna check and throw in dev later